contributors/design-proposals/daemonset-update.md

```diff
@@ -0,0 +1,222 @@
+# DaemonSet Updates
+
+**Author**: @madhusudancs, @lukaszo, @janetkuo
+
+**Status**: Proposal
+
+## Abstract
+
+A proposal for adding the update feature to `DaemonSet`. This feature will be
+implemented on server side (in `DaemonSet` API). 
+
+Users already can update a `DaemonSet` today (Kubernetes release 1.5), which will
+not cause changes to its subsequent pods, until those pods are killed. In this
+proposal, we plan to add a "RollingUpdate" strategy which allows DaemonSet to
+downstream its changes to pods. 
+
+## Requirements
+
+In this proposal, we design DaemonSet updates based on the following requirements:
+
+- Users can have a controlled number of in-flight updates of a DaemonSet (e.g.
```

Comment:
Clarify whether this is "number of simultaneous config changes to a DaemonSet" or "number of simultaneous pods changing to 1 new config".